rest_framework/checks.py

```diff
@@ -19,3 +19,22 @@ def pagination_system_check(app_configs, **kwargs):
             )
         )
     return errors
+
+
+@register(Tags.compatibility)
```

Comment:
This probably should be `Tags.security`, see for example all the checks in https://github.com/django/django/blob/0ee06c04e0256094270db3ffe8b5dafa6a8457a3/django/core/checks/security/base.py

Those checks also use [`deploy=True`, which causes them to only run with `check --deploy`](https://docs.djangoproject.com/en/5.1/topics/checks/#django.core.checks.register). Not sure is this is a good idea. OTOH, it's done the same way for all the other validity checks for security settings.

In any case, am I correct that one has to manually run the `check` command to execute this code (i.e., it's not done on startup)?

Reply:
I went with `Tags.compatibility` largely because the other existing check is also looking to validate settings (which is all my check does, really). `Tags.security` seems to have more to do with actual security checks (rather than mere misconfiguration). Apparently it is also possible to omit the tag entirely.

I'm happy to go with any of these options, just let me know what you prefer.